src/edge/views.py

```diff
@@ -10,20 +11,39 @@
 
 from edge.models import Fragment, Genome, Operation
 from edge.io import IO
+from edge import import_gff
+from edge.tasks import build_genome_blastdb
 
 
 def genome_export(request, genome_id):
     get_genome_or_404(genome_id)
     io = IO(Genome.objects.get(pk=genome_id))
-
     response = HttpResponse(content_type='text/plain')
     response['Content-Disposition'] = "attachment; filename=\"g%s.gff\"" % genome_id
     io.to_gff_file(response)
     return response
 
 
+def genome_import(request):
+    import tempfile
```

Comment:
unless you have a great reason for doing otherwise, it's less confusing IMO to put all imports at the top of a file.